mne/io/ctf/eeg.py

```diff
@@ -49,3 +53,46 @@ def _read_eeg(directory):
                     eeg['np'] += 1
     logger.info('    Separate EEG position data file read.')
     return eeg
+
+
+def _read_pos(directory, transformations):
+    """Read the .pos file and return eeg positions as digitizer extra points.
+    """
+    fname = [join(directory, f) for f in listdir(directory) if
+             f.endswith('.pos')]
+    if len(fname) < 1:
+        return list()
+    elif len(fname) > 1:
+        warnings.warn('    Found multiple pos files. Extra digitizer points '
+                      'not added.')
+        return list()
+    logger.info('    Reading digitizer points from %s...' % fname)
+    if transformations['t_ctf_head_head'] is None:
+        warnings.warn('    No transformation found. Extra digitizer ponts not '
+                      'added')
+        return list()
+    fname = fname[0]
+    digs = list()
+    i = 2000
+    with open(fname, 'r') as fid:
+        for line in fid:
+            line = line.strip()
+            if len(line) > 0:
+                parts = line.decode('utf-8').split()
+                if len(parts) not in [4, 5]:
+                    continue
+                try:
+                    ident = int(parts[0]) + 1000
+                except:
```

Comment:
always `except SOMETHING`, at least `except Exception`. But I think in this case you want `except ValueError`, please add a note it's because the `int` cast could fail